site/content/contribute/more-info/_index.md

```diff
@@ -14,18 +14,17 @@ Here are a few links to Mattermost content that you might find helpful depending
 1. Specific localization guides (coming soon!)
 2. {{<newtabref title="Mattermost contributor agreement" href="https://mattermost.com/mattermost-contributor-agreement/">}}
 3. {{<newtabref title="Approved contributor list" href="https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true">}}
-4. [Making an app]({{<ref "/integrate/apps">}})
-5. {{<newtabref title="The API" href="https://api.mattermost.com">}}
-6. [Incoming]({{<ref "/integrate/webhooks/incoming">}}) and [outgoing]({{<ref "/integrate/webhooks/outgoing">}}) webhooks
-7. [Plugins]({{<ref "/integrate/plugins">}}), also [Plugins]({{<ref "/contribute/more-info/plugins">}})
-8. [Making slash commands]({{<ref "/integrate/slash-commands">}})
-9. [Embedding Mattermost in other places]({{<ref "/integrate/customization/embedding">}})
-10. [Server project]({{<ref "/contribute/more-info/server">}})
-11. [Webapp project]({{<ref "/contribute/more-info/webapp">}})
-12. [Mobile apps]({{<ref "/contribute/more-info/mobile">}})
-13. [Desktop app]({{<ref "/contribute/more-info/desktop">}})
-14. [Focalboard]({{<ref "/contribute/more-info/focalboard">}})
-15. {{<newtabref title="Playbooks" href="https://github.com/mattermost/mattermost-plugin-playbooks">}}
+4. {{<newtabref title="The API" href="https://api.mattermost.com">}}
```

Comment:
Can this be made into the following pattern to avoid superfluous diffs?

```
1.
1.
1.
```

Reply:
If this is a preferred and desired approach, we'd want to apply such a change across the entire docs set rather than to a single page or section. This lift could be a Help Wanted ticket.